addon/components/bs4/bs-navbar.js

```diff
@@ -4,7 +4,19 @@ import Navbar from 'ember-bootstrap/components/base/bs-navbar';
 export default Navbar.extend({
   classNameBindings: ['breakpointClass', 'backgroundClass'],
 
-  type: 'light',
+  type: computed('appliedType', {
+    get() {
+      return this.get('appliedType');
+    },
+
+    set(key, value) { // eslint-disable-line no-unused
+      let newValue = (!value || value === 'default') ? 'light' : value;
+      this.set('appliedType', newValue);
+      return newValue;
+    }
+  }),
+
+  appliedType: 'light',
```

Comment:
`type` is used by the `TypeClass` mixin and I needed somewhere to store the value while giving it a default value. I got tired of wonder why the hamburger menu didn't show when running the demo app in BS4. It's bitten me three times already. It's because it sets `type` to `undefined` if the checkbox is unchecked. Perhaps I shouldn't override the use of `default`, though. What do you think?